src/cryptography/fernet.py

```diff
@@ -134,6 +134,9 @@ def __init__(self, fernets):
     def encrypt(self, msg):
         return self._fernets[0].encrypt(msg)
 
+    def rotate(self, msgs, ttl=None):
+        return [self.encrypt(self.decrypt(m, ttl)) for m in msgs]
```

Comment:
Why does this work on lists of messages? No other fernet APIs do.

Reply:
The main use case I was targeting was rotating an entire set of secrets from one key to another. That said, this could just operate on a single token, and we could leave it up to the consumer to handle larger groups. @alex is that what you'd prefer?